docs/source/porting_guide/PORTING_2.rst

```diff
@@ -0,0 +1,62 @@
+------------------------
+Receptor 2 Porting Guide
```

Comment:
Is "2" the version number here or does it indicate something else? "2.x" might make that a little more obvious.

Reply:
Reasonable, updated